ext/native/gfx_es2/gpu_features.cpp

```diff
@@ -418,6 +418,15 @@ void CheckGLExtensions() {
 	} else {
 		g_all_egl_extensions = "";
 	}
+#elif defined(USING_GLES2) && defined(__linux__)
```

Comment:
Maybe this should just be `#elif defined(USING_EGL)`?

-[Unknown]

Reply:
No, it wouldn't work then if SDL2 is doing EGL Init which is normally the case when GLES2 is true and SDL2 is used.

i.e on ODROID's we explicitly disable it -DUSING_EGL=OFF because we have SDL2 doing EGL Init

what I am not sure about is if this would also work on Apple so we could ad defined(__APPLE__) too